core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -49,25 +60,98 @@
 // -----------------------------------------
 
 .searchbar-clear-button {
-  @include globals.position(50%, globals.$ionic-space-200, null, null);
+  @include globals.position(
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    null,
+    null
+  );
 
-  position: absolute;
+  width: searchbarVars.$searchbar-ionic-input-clear-icon-size;
+  height: searchbarVars.$searchbar-ionic-input-clear-icon-size;
 
-  width: globals.$ionic-scale-600;
-  height: globals.$ionic-scale-600;
+  background-color: transparent;
 
-  transform: translateY(-50%);
+  font-size: globals.$ionic-font-size-400;
+
+  contain: strict;
+}
+
+// Searchbar Cancel Icon
+// -----------------------------------------
+
+.searchbar-cancel-button {
+  /**
+   * The left edge of the cancel button
+   * should align with the left edge
+   * of the back button if the searchbar
+   * is used in a toolbar.
+   */
+  @include globals.position(0, null, null, 9px);
 
   background-color: transparent;
 
   font-size: globals.$ionic-font-size-400;
+}
 
-  contain: strict;
+// Searchbar Search & Clear Icon & Cancel Icon
+// -----------------------------------------
+
+.searchbar-search-icon,
+.searchbar-clear-button,
+.searchbar-cancel-button {
+  position: absolute;
+}
+
+// Clear Icon & Cancel Icon
+// -----------------------------------------
+
+.searchbar-clear-button:focus-visible,
+.searchbar-cancel-button:focus-visible ion-icon {
+  @include globals.border-radius(globals.$ionic-border-radius-100);
+
+  outline: globals.$ionic-border-size-050 globals.$ionic-border-style-solid globals.$ionic-state-focus-1;
+
+  opacity: 1;
 }
 
 // Searchbar in Toolbar
 // -----------------------------------------
 
 :host-context(ion-toolbar) {
-  min-height: globals.$ionic-scale-1000;
+  min-height: globals.$ionic-scale-1200;
+}
+
+// Searchbar States
+// --------------------------------------------------
+
+/* Hover */
+:host(:hover) {
+  --icon-color: #{globals.$ionic-color-neutral-900};
```

Comment:
I am seeing `color/neutral/800` in Figma on hover, is this right?

Reply:
You're right. It seems that the only thing that changes is the background: https://github.com/ionic-team/ionic-framework/pull/29917/commits/127027c47560cdff89cc20d685bd250516e9907a